### ISSUE TITLE
Handle interface with Date property

### ISSUE THREAD

**Issue by karlbohlmark** [juusaw/ts-to-io#5]
Hi, thanks for a potentially very useful lib!

Currently, given an interface with a date property
```
export interface MyInterface {
    myDate: Date;
}
```

**Comment by juusaw** [comment]
Hi,

As far as I know there's no default codec available in io-ts for `Date` objects. ts-to-io could generate a custom codec for them. However, I haven't so far decided whether to extend the project with custom codecs (which would also enable support for codecs for tuple types with rest parameters) but I'll look into it some time in the near future. One option would be to add a config flag for whether to include custom codecs if a processed type would require one.

**Comment by karlbohlmark** [comment]
Yes, after looking a bit closer at io-ts I understand that is the reason. It feels kind of lacking to not be able to handle dates though. That's the only 'codec' type of thing I need, otherwise I just want to use io-ts for validation. I guess DateFromISOString from https://github.com/gcanti/io-ts-types is a reasonable custom codec to use.

**Comment by ahrjarrett** [comment]
Let's close this one? Supporting `io-ts-types` seems outside the scope of this project
